joomla-extensions/patchtester
# Support non CMS like Repo Struktures

#### Summary of Changes

With this change we are able to apply patches also for non CMS like Repo Struktures.
The requirement is that the files are placed inside of the `src` folder and follow than the CMS Strukturs.

This PR also adds:
- some more files to the exclude array e.g. from the patchtester & weblinks repo.
- and weblinks as supported extension / repo

#### Testing Instructions

- install patchtester
- instal weblinks: https://github.com/joomla-extensions/weblinks/releases/download/3.5.0/pkg-weblinks-3.5.0.zip
- switch in the patchtester config to custom and enter: `joomla-extensions` & `weblinks`
- fetch the data
- apply a PR (e.g. the CS PR from yesterday)
- it fails.
- apply this patch to the patchtester
- try to apply the weblinks patch again.
- it should work
- revert the patch
- it should work
